README.md

```diff
@@ -25,12 +25,41 @@ Composer will install the library to your project's `vendor/goaop/parser-reflect
 Usage
 ------------
 
+### Initialization
+
+Prior to first use library needs can optionally be initialized.
+
+Following code will use Composer's autoloader of project, where library is used as a dependency:
```

Comment:
Maybe we can reword? Future tense is not suitable well for describing processes in source code, simple present tense is more natural: 
```
Following example uses the Composer autoloader from the project where library is used as a dependency:
```